api/dish.js

```diff
@@ -39,14 +39,14 @@ function getManyDishes(condition = {}) {
 }
 
 async function createDish(req) {
-  let { name, description, businessId, price, isHidden } = req.body;
+  let { name, description, businessId, price, isHidden, language } = req.body;
```

Comment:
It will be a bit ugly to add this param to all the endpoints.
Maybe we can just append it to the `req` in a generic way for all endpoints, and then do `req.language` or something like this?

Reply:
I added it as a header to every request so it'll be under req.headers.language